packages/solidity-contracts/contracts/test/PermitToken.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.9;
+
+import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import {ERC20Permit} from "@openzeppelin/contracts/token/ERC20/extensions/ERC20Permit.sol";
+
+/// @notice This token is for testing purposes.
+contract PermitToken is ERC20, ERC20Permit {
```

Comment:
nit: can be called MockPermitToken to avoid confusion?

Reply:
hm, there is standard erc20 `Token`  contract we use for unit tests, so went with that convention